mlrun/serving/server.py

```diff
@@ -307,15 +308,33 @@ def wait_for_completion(self):
         """wait for async operation to complete"""
         return self.graph.wait_for_completion()
 
+    @staticmethod
+    def get_serving_spec():
+        data = None
+
+        try:
+            with open(mlrun.common.constants.MLRUN_SERVING_SPEC_PATH) as f:
+                data = f.read()
+        except FileNotFoundError:
+            # we will have the serving spec in either mounted config map
+            # or env depending on the size of the spec and configuration
```

Comment:
That's right, but inside the `except` we already know that it is not a mounted config map